### ISSUE TITLE
Different icon for debug versions

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#474]
### Describe your idea

As we are able to install debug versions in parallel of the release version, it would be good to have a different icon (for example, a red background instead) so we can easily distinguish both.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to a question covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document, or a feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[version 4.6.6 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.
